### PR TITLE
Updates IL incentives based on feedback received from Elevate

### PR DESCRIPTION
**Pull request by matelau**
Adds Incentive programs based on feedback we received from Elevate. For more details see Asana [ticket](https://app.asana.com/0/0/1207048430600242/f). 

TODO: 
- Submit ticket for updating translations for new content will do once the content is reviewed here.

### CODE REVIEW

File: data/IL/incentives.json [hunk 1]
```diff
@@ -631,28 +720,231 @@
...
+    "program": "il_evanstonDevelopmentCooperative_evanstonGreenHomesPilotProgram",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $20,000 for energy effiency, and health improvement updates. In most cases, the program will cover the full cost of home improvements."
```
Comment (oyamauchi) [inline data/IL/incentives.json:883]: `efficiency` is misspelled

File: data/IL/incentives.json [hunk 2]
```diff
@@ -631,28 +720,231 @@
...
```
Comment (oyamauchi) [inline data/IL/incentives.json:904]: Same "efficiency" typo, and this description doesn't match the item

File: data/IL/incentives.json [hunk 3]
```diff
@@ -631,28 +720,231 @@
...
```
Comment (oyamauchi) [inline data/IL/incentives.json:946]: Same typo, and doesn't match item

File: src/data/programs/il_programs.ts [hunk 4]
```diff
@@ -1,42 +1,106 @@
...
-  il_stateOfIllinois_homeWeatherization: {
+  il_cornBeltEnergy_rebateProgram: {
     name: {
-      en: 'Illinois Home Weatherization Assistance Program',
+      en: 'Rebate Program',
+    },
+    url: {
+      en: 'https://www.cornbeltenergy.com/programs-services/rebate-program',
+    },
+  },
```
Comment (oyamauchi) [inline src/data/programs/il_programs.ts:41]: Is this still used? It's weird to me that there's both this one and the one above

File: test/routes/v1.test.ts [hunk 5]
```diff
@@ -283,6 +283,23 @@ test('IL low income response with state and utility filtering is valid and corre
...
+    t,
+    {
+      zip: '60202',
+      owner_status: 'homeowner',
+      household_size: 2,
+      household_income: 10000,
+      tax_filing: 'single',
+      authority_types: ['city'],
+      // TODO: Remove when IL is fully launched.
+      include_beta_states: true,
```
Comment (oyamauchi) [inline test/routes/v1.test.ts:297]: IL is fully launched, so we can delete this

**Review by oyamauchi — APPROVED** [review]
Just a few minor things

File: src/data/programs/il_programs.ts (hunk 4 above)
Comment (matelau) [reply src/data/programs/il_programs.ts:41]: Looks like there was one rogue row in the spreadsheet that need the program title updated - fixed ✅

**Pull request merged**